Library/Homebrew/brew.rb

```diff
@@ -35,6 +35,10 @@ class MissingEnvironmentVariables < RuntimeError; end
   exec ENV["HOMEBREW_BREW_FILE"], *ARGV
 end
 
+def head_unsupported_error
+  $stderr.puts "HEAD builds are unsupported by maintainers - please file pull requests instead of issues."
```

Comment:
```suggestion
  $stderr.puts <<~EOS
    Please create pull requests instead of asking for help on Homebrew's GitHub, 
    Discourse, Twitter or IRC.
  EOS
```

Reply:
`diagnostic.rb` has `please_create_pull_requests` which may be useful.

Reply:
Yeh, that could be pulled into `Utils` somewhere instead 👍